README.md

```diff
@@ -237,7 +237,9 @@ We are using [lerna to publish](https://github.com/lerna/lerna/tree/master/comma
 
 - Create a branch for releasing。 `git branch release`
 - `git push release` and set the upstream.
+- `npm config delete @tradeshift:registry` delete the registry config so release could publish to different registries
 - `npm run release`
+- `npm config set @tradeshift:registry https://npm.pkg.github.com/` Set back npm to use github for @tradeshift packages
```

Comment:
I was wondering is it possible to merge these step to release script?

Reply:
I think it would be better to do in a separate shell script, because it would be nice to check this value first, change it only if it exists, and return to the initial state after release.